.github/workflows/commit.yaml

```
@@ -47,6 +47,7 @@ jobs:
         with:
           path: |
             ~/.cache/go-build
+            ~/.cache/golangci-lint
             ~/go/pkg/mod
             ~/go/bin
           key: check-${{ runner.os }}-go-${{ matrix.go-version }}-${{ hashFiles('**/go.sum', 'Makefile') }}
```

could you change the key a bit to see if it's working correctly? 
```suggestion
          key: check-${{ runner.os }}-go-${{ matrix.go-version }}-${{ hashFiles('**/go.sum', 'Makefile') }}-
```

---

ah ok nvm! path change actually causes the invalidation